source/simulator/solver_schemes.cc

```diff
@@ -291,8 +291,9 @@ namespace aspect
         const double relative_stokes_residual =
           assemble_and_solve_stokes(nonlinear_iteration == 0, &initial_stokes_residual);
 
-        pcout << "      Relative Stokes residual after nonlinear iteration " << nonlinear_iteration+1
+        pcout << "      Relative  nonlinear residual (Stokes system) after nonlinear iteration " << nonlinear_iteration+1
```

Comment:
one space too much between Relative and nonlinear

Reply:
Thanks for catching that!